zarr/tests/test_storage.py

```diff
@@ -1020,6 +1020,7 @@ def create_store(self, normalize_keys=False, dimension_separator=".", **kwargs):
         )
         return store
 
+    @pytest.mark.xfail(reason="Skip under Pyodide for now, don't know how to handle umask")
```

Comment:
This is one of the tests I would like to receive help on – I am not entirely aware of the differences in the Pyodide (Emscripten MemFS to be specific) and the CPython file systems, and I see that retrieving the 0o777 octal (execute permissions) seems to bring some insights. I have marked this as xfail for now.